bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/NativeInterface.cs

```diff
@@ -1946,6 +2018,27 @@ protected override bool Free()
                 ElementModQ.ElementModQHandle manifest_hash,
                 out CiphertextElectionContextHandle handle);
 
+            [DllImport(DllName, EntryPoint = "eg_ciphertext_election_context_make_with_extended_data",
+                CallingConvention = CallingConvention.Cdecl, SetLastError = true)]
+            internal static extern Status Make(
+                ulong number_of_guardians,
+                ulong quorum,
+                ElementModP.ElementModPHandle elgamal_public_key,
+                ElementModQ.ElementModQHandle commitment_hash,
+                ElementModQ.ElementModQHandle manifest_hash,
+                LinkedList.LinkedListHandle extended_data,
```

Comment:
so here in the make function we can also access the liked list handle type so that we know we are passing in an object of the correct pointer type.